app/models/appeals/AppealSubmission.scala

```diff
@@ -431,90 +457,110 @@ object AppealSubmission {
     reasonableExcuse match {
       case "bereavement" => {
         AppealSubmission(
-          submittedBy = if (userRequest.isAgent) "agent" else "client",
-          reasonableExcuse = reasonableExcuse,
-          honestyDeclaration = userRequest.session.get(SessionKeys.hasConfirmedDeclaration).get == "true",
+          sourceSystem = "MDTP",
+          taxRegime = "VAT",
+          customerReferenceNo = s"VRN${userRequest.vrn}",
+          dateOfAppeal = LocalDateTime.now(),
+          isLPP = !userRequest.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Submission.toString),
```

Comment:
Could this be replaced with:
```suggestion
          isLPP = userRequest.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Payment.toString),
```